grants_tagger/utils.py

```diff
@@ -1,14 +1,15 @@
 # encoding: utf-8
-from functools import partial
 import pickle
 import json
 
 import requests
-from sklearn.model_selection import train_test_split
 from sklearn.metrics import f1_score
 import tensorflow as tf
 import pandas as pd
 import numpy as np
+from functools import partial
```

Comment:
do you use that?

Reply:
actually you just moved them it seems. is that black or you?

Reply:
Honestly don't know, I think it was me cause at one point I had moved part of it in another file but then decided to put it back. 
WIll put them back in the same order they were.